README.md

```diff
@@ -1,167 +1,169 @@
-<br />
-<br />
-• Manage dedicated [Northstar](https://northstar.tf) servers.
-<br /><br />
+## Features:
 
-_Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new] and fill out the fields._
-<br />
-<br /><br />
+* Easilly install, update and launch [Northstar](https://northstar.tf)
+
+* Manage installed [Northstar](https://northstar.tf) mods
+
+* Downloading and installing [Northstar](https://northstar.tf) mods from a GitHub/GitLab repository
+
+* Installing downloaded [Northstar](https://northstar.tf) mods (.zip files)
+
+* Easilly isntall custom Wepaon/Pilot Skins
+
+* Easilly start a [Northstar](https://northstar.tf) Dedicated Server
+
+* [Thunderstore](https://northstar.thunderstore.io) Mod Browser
+
+
+### WIP Features:
+
+* Create custom servers using this application to fine tune setups.
+
+* Manage dedicated [Northstar](https://northstar.tf) servers.
+
+### Suggestions/Bug reports?
+* Suggestions/bug reports are highly appreciated. Please create a new [issue](https://github.com/BigSpice/VTOL/issues/new) and fill out the fields.
+
+### Installation and updates
+* Install this application by downloading and running the latest [release from here](https://github.com/BigSpice/VTOL/releases)
+   * ``Download the .msi package``
 
-Install this application by downloading and running the latest release:
-<br /><br />
 ![image](https://user-images.githubusercontent.com/23240514/150025911-aacaf334-cc33-4239-94d7-512cfb84fe49.png)
-<br /><br />
-If you already use this application, consider navigating to "check for updates" inside the application to download and install the latest release.<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447226-88662295-b7a9-4fd5-a958-dfc5ce59bd9c.png)<br /><br />
-Once the application successful downloaded and installed:
-<br /><br />
-<br /><br /><br />
-
-1. Configure your Titanfall 2 installation directory.
-   <br /><br />
-   If the automatic browse cannot find your Titanfall 2 installation, please select "browse" and _browse_ to where your Titanfall 2 installation is, and select it.
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446386-4bb584c1-038a-4dd6-9588-ac3827884144.png)<br /><br />
-2. Verify, or install the [Northstar Client](https://northstar.tf)
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446412-d10f1f7e-bbc2-4558-8e8b-033aae2af410.png)<br /><br />
-   Use this to verify your existing Northstar installation.
-   <br /><br />
-   _By using that, the application will download the latest [Northstar release](https://github.com/R2Northstar/Northstar/releases/latest)._
-   <br /><br />
-   ![image](https://user-images.githubusercontent.com/23240514/152446466-26d91f33-948d-4051-9247-c21b20328515.png)
 
-<br /><br />
+* This application has an auto-update Feature in the Update section
```

Comment:
the wording is a bit off here

Reply:
What do you suggest I put there instead?

Reply:
`You can preferably enable the automatic-updater module in the \`Update\` section`

Reply:
updated